addons/l10n_in_edi_ewaybill/views/account_move_views.xml

```diff
@@ -19,20 +19,20 @@
                             <field name="l10n_in_type_id" domain="[('allowed_supply_type', 'in', ('in', 'both')) if move_type in ('in_invoice', 'in_refund', 'in_receipt') else ('allowed_supply_type', 'in', ('out', 'both'))]"/>
                         </group>
                         <group string="Transportation Details" name="transportation_group">
-                            <field name="l10n_in_mode"/>
+                            <field name="l10n_in_mode" widget="radio"  options="{'horizontal': True}"/>
                             <label for="l10n_in_distance"/>
                             <div class="o_row" name="l10n_in_distance">
-                                <field name="l10n_in_distance"/>
+                                <field name="l10n_in_distance" class="oe_inline"/>
                                 <span>km</span>
                             </div>
-                            <field name="l10n_in_vehicle_type" 
+                            <field name="l10n_in_vehicle_type" widget="radio" options="{'horizontal': True}"
                                 attrs="{'invisible': [('l10n_in_mode','!=','1')], 'required': [('l10n_in_mode','=','1')]}"/>
                             <field name="l10n_in_vehicle_no" 
                                 attrs="{'invisible': [('l10n_in_mode','!=','1')], 'required': [('l10n_in_mode','=','1')]}"/>
                             <field name="l10n_in_transporter_id"
                                 domain="[('vat','not in', ['', False]), ('country_id.code','=','IN')]"
                                 attrs="{'invisible': [('l10n_in_mode','!=','0')], 'required': [('l10n_in_mode','=','0')]}"/>
-                            <field name="l10n_in_transportation_doc_no" 
+                            <field name="l10n_in_transportation_doc_no" string="Transporter Document Number"
```

Comment:
You could also change it in the field definition.  (I suppose it is not only in this view that we would want it to be called like that)